docs/guides/expectations/create_expectations_overview.md

```diff
@@ -3,4 +3,67 @@ title: "Create Expectations: Overview"
+
+The second workflow is the automated work flow.  In this workflow, you will use a Profiler to generate Expectations based on some input data.  The Validator will then save that those Expectations as an Expectation Suite in an Expectation Store.  You will also have the option of vieiwing them in your Data Docs.  A more thorough overview of this workflow, and a link to an in depth guide on it, can be found in this document's section on [creating Expectations automatically](#creating-expectations-automatically).
+
+The third workflow, which is not recommended, is to manually define your Expectations by writing their configurations.  This workflow does not require source data to work against, but does require a deep understanding of the configurations available for Expectations.  We will forgo discussion of it in this document, and focus on the two recommended workflows.  If for some reason you must use this workflow, we do provide an in depth guide to it in our documentation on [how to create and edit expectations based on domain knowledge without inspecting data directly](./how_to_create_and_edit_expectations_based_on_domain_knowledge_without_inspecting_data_directly.md).
+
+Regardless of which of the first two workflows you have followed, once you have saved your Expectation Suite it is advised that you test them by validating them against the batch or batches of data you created them against.  This process is the same in either workflow, since at this point you will be using a saved Expectation Suite and each of the prior workflows ends with the saving of your Expectation Suite.  Instructions for this will be detailed in this document's section on [testing your Expectation Suite](#testing-your-expectation-suite).
+
+### Creating Expectations interactively
+
+When using the interactive method of creating Expectations, you will start as you always do with your Data Context.  In this case, you will want to navigate to your Data Context's root directory in your terminal, where you will use the CLI to launch a Jupyter Notebook which will contain scaffolding to assist you in the process.  You can even provide flags such as `--profile` which will allow you to enter into the interactive workflow after using a Profiler to generate and prepopulate your Expectation Suite.
+
+We provide an in depth guide to using the CLI (and what flags are available to you) for interactively creating Expectations in our guide on [how to create and edit Expectations with instant feedback from a sample batch of data](./how_to_create_and_edit_expectations_with_instant_feedback_from_a_sample_batch_of_data.md).
```

Comment:
This led me down a rabbit hole! It looks like "in-depth" is an adjective, while "in depth" is an adverb, so I think you want the former here (and in a few other places in the doc).

Reply:
Correction made.  I think I'm going to have to add something as a "Documentation standards specific to me" to check for that throughout the docs, because it is likely I've overlooked it elsewhere.